src/client/sandbox/node/element.js

```diff
@@ -29,6 +30,10 @@ export default class ElementSandbox extends SandboxBase {
         this.BEFORE_FORM_SUBMIT = 'hammerhead|event|before-form-submit';
     }
 
+    static _isNamedTarget (value) {
+        return value[0] !== '_';
+    }
```

Comment:
`_isNamedTarget` is a unclear function name.
It checks that target is not in reserved words - `_self`, `_parent`, `_top`.
Maybe rename it to `_isBuildInTarget` or `_isReservedTarget`.
@churkin What do you think?